priv/repo/seeds/20201230073325_v0_7_0_alter_glific_data.exs

```diff
@@ -0,0 +1,39 @@
+defmodule Glific.Repo.Seeds.AddGlificData_v0_7_0 do
+  use Glific.Seeds.Seed
+
+  envs([:dev, :test, :prod])
+
+  alias Glific.{
+    Partners.Provider,
+    Partners.Credential,
+    Repo
+  }
+
+  def up(_repo) do
+    update_existing_credentials()
+    delete_provider()
+  end
+
+  defp update_existing_credentials() do
+    credentials = Repo.all(Credential, skip_organization_id: true)
+
+    credentials
+    |> Enum.each(fn credential ->
+      if Enum.member?([3, 5], credential.provider_id) do
```

Comment:
Same here?

Reply:
@pankaj-ag we are checking provider_id in credentials here. So for using shortcut we might need to preload here. Shall I do that?